app/src/main/res/layout/activity_view_binding_adapters_test.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+  <FrameLayout
+    android:id="@+id/expand_list_icon"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="8dp"
+    android:layout_marginEnd="8dp"
+    android:layout_marginBottom="8dp"
+    >
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Done.